### DISCUSSION TITLE
BGM fade parameter for stages

### DISCUSSION THREAD

**Discussion by UrsoManso** [ikemen-engine/Ikemen-GO#1516] [Feature Request]
### Is your feature request related to a problem? Please describe.

https://youtu.be/xI284D4y1q4?t=95

Similar to SF2 where the music fades out after the round/match ends

### Describe the solution you'd like

Two boolean parameters under the stage's music section, example:

For rounds
BGM.fade.ko =

For matches
BGM.fade.win =

**Comment by Sluigi123** [comment]
I remember the Street Fighter Alpha games have this feature too! It'll be a nice addition to IKEMEN as well!
